packages/web-service/src/services/authorization.js

```diff
@@ -1,11 +1,16 @@
 import { LOGIN } from '../uris.js'
 
 export default (_server, _options) => ({
-  authenticate: async (request, h) => {
+  authenticate: async function (request, h) {
```

Comment:
I'm sure there's a reason for changing this to a `function` but I can't immediately see what it is - a short comment might be helpful, to avoid anyone accidentally changing it again later.

Reply:
I have added a comment.

Reply:
It preserves the 'this' pointer, which I am not using explicitly. (I know this was needed in the request decorator but ATM I am not sure why I used it here.

Reply:
Yeah I couldn't see any mention of `this` which is why I was confused